An error occurred while executing the following cell:
------------------
import cv2
import jax
import augmax
from jax import numpy as jnp
from jax import random as jrand
from skimage.util import montage

import tensorflow_datasets as tfds

import matplotlib.pyplot as plt

%matplotlib inline
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_4644/3866412800.py in <module>
      6 from skimage.util import montage
      7 
----> 8 import tensorflow_datasets as tfds
      9 
     10 import matplotlib.pyplot as plt

ModuleNotFoundError: No module named 'tensorflow_datasets'
ModuleNotFoundError: No module named 'tensorflow_datasets'
